src/Google.Api.Gax/Api/Gax/ServiceEndpointSettings.cs

```diff
@@ -5,6 +5,7 @@
  * https://developers.google.com/open-source/licenses/bsd
  */
 
+using Grpc.Core;
```

Comment:
Yup. I'll squash to one commit and remove any straggling bits like this or whitespace.